libbeat/processors/actions/drop_fields.go

```diff
@@ -62,25 +67,49 @@ func newDropFields(c *conf.C) (processors.Processor, error) {
 		}
 	}
 
-	f := &dropFields{Fields: config.Fields, IgnoreMissing: config.IgnoreMissing}
+	// Parse regexp containing fields and removes them from initial config
+	regexpFields := make([]match.Matcher, 0)
+	for i := len(config.Fields) - 1; i >= 0; i-- {
+		field := config.Fields[i]
+		if strings.HasPrefix(field, "/") && strings.HasSuffix(field, "/") && len(field) > 2 {
+			config.Fields = append(config.Fields[:i], config.Fields[i+1:]...)
+
+			regexpFields = append(regexpFields, match.MustCompile(field[1:len(field)-1]))
```

Comment:
Sorry, just noticed this. `match.MustCompile` will panic if the value is invalid, that's not what we want, it should return an error like the one above on line 57.

Reply:
👍 done